src/topology/metric_space/basic.lean

```diff
@@ -378,6 +378,17 @@ assume y (hy : _ < _), le_of_lt hy
 theorem sphere_subset_closed_ball : sphere x ε ⊆ closed_ball x ε :=
 λ y, le_of_eq
 
+lemma ball_disjoint_ball (x y : α) (rx ry : ℝ) (h : rx + ry ≤ dist x y) :
+  disjoint (ball x rx) (ball y ry) :=
```

Comment:
Is this true as an `iff`?

Reply:
No. For instance, in the subspace `{0, 4}`of the real line, the balls of radius `3` centered at these two points are disjoint, and still the sum of the radii is `> 4`.